plugins/modules/swim_intent.py

```diff
@@ -741,8 +741,8 @@ def get_device_uuids(self, site_name, device_family, device_role, device_series_
 
         device_uuid_list = []
         if not site_name:
-            self.log("Failed to retrieve devices associated with the site due to missing site name", "INFO")
-            return device_uuid_list
+            site_name = "Global"
+            self.log("Since site name is not given so it will be fetch all the devices under Global and mark site name as 'Global'", "INFO")
```

Comment:
Site name not specified; defaulting to 'Global' to fetch all devices under this category

Reply:
updated.